lib/router/selectedProduct.js

```diff
@@ -1,12 +1,12 @@
-FlowRouter.route('/ProductsExample', {
+FlowRouter.route('/SelectedProduct', {
```

Comment:
We might want to think about making the path '/Product' rather than '/SelectedProduct'. It's just more concise. All of the variable names everywhere that are named selectedProduct are good though because it ensures we know what it is.